chat-plugins/info.js

```diff
@@ -1785,7 +1785,7 @@ exports.commands = {
 		if (!height) return this.errorReply('No height for the image was provided!');
 		if (!isNaN(height)) height += 'px';
 
-		let unitRegex = /^\d+(?:p[xtc]|%|[ecm]m|ex|in)$/;
+		let unitRegex = /^\d+(?[xtc]|%|[ecm]m|ex|in)$/;
```

Comment:
I interpreted it as "I saw travis was failing due to a change of a regex, but I didn't change that...", so I pointed it out specifically just in case